decoration.sk

```diff
@@ -595,4 +595,51 @@ buzzy bee update:
 	{directional} {honey level} bee nest¦s = minecraft:bee_nest
 
 	honey block¦s = minecraft:honey_block
-	honeycomb block¦s = minecraft:honeycomb_block
\ No newline at end of file
+	honeycomb block¦s = minecraft:honeycomb_block
+
+nether update:
+	minecraft version = 1.16 or newer
+
+	shroomlight¦s = minecraft:shroomlight
+	[floor] soul torch¦es = minecraft:soul_torch
+	{directional} soul wall torch¦es = minecraft:soul_wall_torch
+	weeping vine¦s = minecraft:weeping_vines
+	weeping vine (block|plant)¦s = minecraft:weeping_vines_plant
+	twisting vine¦s = minecraft:twisting_vines
+	twisting vine (block|plant)¦s = minecraft:twisting_vines_plant
+	nether sprout¦s = minecraft:nether_sprouts
+	{waterloggable} chain¦s = minecraft:chain
+	lodestone¦s = minecraft:lodestone
+
+	{nether block} fung(us|i) = -fungus
+	{nether block} root¦s = -roots
+
+	[any] fung(us|i) = crimson fungus, warped fungus
```

Comment:
Up to you, but to me "any fungus" sounds like it'd include the old red/brown mushrooms as well. Maybe this should be `[any] nether fung(us|i)`?

Reply:
Fixed